src/install/db.py

```diff
@@ -63,7 +63,7 @@ def main():
 
     with OperateInDirectory('../../'):
         with suppress(FileNotFoundError):
-            Path('start_fact_db').unlink()
-        Path('start_fact_db').symlink_to('src/start_fact_db.py')
+            Path('start_fact_database').unlink()
+        Path('start_fact_database').symlink_to('src/start_fact_database.py')
```

Comment:
renaming this could mean that you have two files `start_fact_db` and `start_fact_database` after you run this in an existing FACT instance (which could be confusing and could even lead to errors if the user keeps using the old one).

Reply:
I don't see a problem here.
Mark this as a breaking change in the changelog and users are good to go.
Also if someone uses FACT they probably should be competent enough to realise that the symlink is invalid and then recognise that there is a new file which has almost the same name.

Reply:
> Mark this as a breaking change in the changelog and users are good to go.

So you expect that all users actually read that? :laughing: I just want to avoid reasons for new issues if possible.

We could turn it into a "non-breaking" (or at least "less breaking"?) change if we also try to remove `start_fact_db` in addition to the new file. This would avoid any inconsistencies upfront. I'm not opposed to renaming the file.